### PR TITLE
Add `Project Loved: December 2022` news post

### PR DESCRIPTION
**Pull request by Hiviexd**
scheduled for the 15th

TODO:
- [x] desc reviews pre-PR
- [x] confirm date
- [x] generate forum posts

### CODE REVIEW

**Review by Walavouchey — DISMISSED** [review]
we have slightly more people reviewing descriptions nowadays

File: news/2022-12-15-project-loved-december-2022.md [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+- [Results for osu!taiko](https://osu.ppy.sh/community/forums/posts/1675561)
+- [Results for osu!catch](https://osu.ppy.sh/community/forums/posts/1675560)
+- [Results for osu!mania](https://osu.ppy.sh/community/forums/posts/1675559)
+
+Before we move on with this month's picks, let's talk about some changes within the Project Loved team:
+
+- We'd like to bid farewell to [Noffy](https://osu.ppy.sh/users/1541323), [Nokashi](https://osu.ppy.sh/users/5431196), and [Zeus-](https://osu.ppy.sh/users/5464437), as they've resigned from the team very recently. Thank you for making Loved awesome by making sure that all picks have been thoroughly checked and prepared for the Loved category in terms of integral metadata and moderation checks!
+- On the other hand, say hello to [Maxus](https://osu.ppy.sh/users/4335785)! He will be joining the metadata team to make sure that picks are in pristine condition before receiving their well-earned leaderboards.
+- A new drummer approaches! [Eriha](https://osu.ppy.sh/users/16320311) joins the osu!taiko Loved captains in order to help with bringing some of the community's most adored beatmaps to justice by granting them all a chance in the Loved polls.
```
Comment (aceticke) [inline news/2022-12-15-project-loved-december-2022.md:24]:
```suggestion
- A new drummer approaches! [Eriha](https://osu.ppy.sh/users/16320311) joins the osu!taiko Loved captains in order to help bring some of the community's most adored beatmaps to justice by granting them all a chance in the Loved polls.
```

File: news/2022-12-15-project-loved-december-2022.md [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+
+That should be all in terms of updates for this months. Don't forget if you're interested in participating in Project Loved discussions of a certain game mode, or have any questions, that you can reach out in one of the Project Loved Discord servers listed below:
```
Comment (aceticke) [inline news/2022-12-15-project-loved-december-2022.md:26]:
```suggestion
That should be all in terms of updates for this month. Don't forget if you're interested in participating in Project Loved discussions of a certain game mode, or have any questions, that you can reach out in one of the Project Loved Discord servers listed below:
```

File: news/2022-12-15-project-loved-december-2022.md [hunk 3]
```diff
@@ -0,0 +1,372 @@
...
+
+Created in collaboration with [Pengdoll](https://osu.ppy.sh/users/6392061), this has mapping styles you would typically see from both of them. We captains aim to kill two birds with one stone with this nomination, and we feel that this delivers some of the best charting from both creators here. Typical to LN charts like these, you'll see a lot of notes in between columns occupied by LNs, which is great for increasing your finger strength overall.
+
+Aside from the above, we know that many people in the community are fans of YOASOBI's music, so we know those fans will be happy to see another YOASOBI chart in the Loved voting. If you want to see a good LN collaboration practice chart in Loved, or you just want to jam to YOASOBI, remember to give *Ano Yume o Nazotte* your support!
+
+---
+
+From [everyone working on Project Loved](https://loved.sh/contributors), we hope you enjoyed this month's selection, and are looking forward to the next! The favoured beatmaps will hit the [Loved beatmap listing](https://osu.ppy.sh/beatmapsets?s=loved) shortly after the polls end. So don't forget to stay hydrated and vote on your favourite beatmaps in order for that to happen!
```
Comment (aceticke) [inline news/2022-12-15-project-loved-december-2022.md:366]:
```suggestion
From [everyone working on Project Loved](https://loved.sh/contributors), we hope you enjoyed this month's selection and are looking forward to the next! The favoured beatmaps will hit the [Loved beatmap listing](https://osu.ppy.sh/beatmapsets?s=loved) shortly after the polls end. So don't forget to stay hydrated and vote on your favourite beatmaps in order for that to happen!
```

**Review by aceticke — CHANGES_REQUESTED** [review]
took a peek, made some suggestions

File: news/2022-12-15-project-loved-december-2022.md [hunk 4]
```diff
@@ -0,0 +1,372 @@
...
+
+[![](/wiki/shared/news/2022-12-15-project-loved-december-2022/322808.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[LIQU@. - Yotogibanashi no Kamikakushi](https://osu.ppy.sh/beatmapsets/322808#osu) by [Ekoro](https://osu.ppy.sh/users/284905)\
+240 BPM, 1:48 | 8.40★\
+*description written by [Skill](https://osu.ppy.sh/users/7802027)*
+
+Ever thought about a Time Freeze II..? Yes? Well, it's here in December's voting.
+
+\[Ekoro](https://osu.ppy.sh/users/284905)'s take on LIQU@.'s *Yotogibanashi no Kamikakushi* is very reminiscent of his own very popular map [UNDEAD CORPORATION - Everything will Freeze [Time Freeze\]](https://osu.ppy.sh/beatmapsets/158023#osu/555797). Just like Time Freeze, this map features 240 BPM death/spaced streams, along with the iconic squares littered throughout the map. The main difference is that this song only sits at 1:48 in length. Plenty of room for some retrying, and world class score potential!
```
Comment (aceticke) [inline news/2022-12-15-project-loved-december-2022.md:120]:
```suggestion
[Ekoro](https://osu.ppy.sh/users/284905)'s take on LIQU@.'s *Yotogibanashi no Kamikakushi* is very reminiscent of his own very popular map [UNDEAD CORPORATION - Everything will Freeze [Time Freeze\]](https://osu.ppy.sh/beatmapsets/158023#osu/555797). Just like Time Freeze, this map features 240 BPM death/spaced streams, along with the iconic squares littered throughout the map. The main difference is that this song only sits at 1:48 in length. Plenty of room for some retrying, and world class score potential!
```

Comment (aceticke) [reply news/2022-12-15-project-loved-december-2022.md:120]: Seems the backslash broke the link

File: news/2022-12-15-project-loved-december-2022.md [hunk 5]
```diff
@@ -0,0 +1,372 @@
+---
+layout: post
+title: "Project Loved: December 2022"
+date: 2022-12-15 18:00:00 +0000
+---
+
+Celebrate Christmas with some festive Loved gifts! 🎄🎁
```
Comment (Blooshing) [inline news/2022-12-15-project-loved-december-2022.md:7]:
``` Suggestion
Celebrate the holidays with some festive Loved gifts! 🎄🎁
```

File: news/2022-12-15-project-loved-december-2022.md [hunk 6]
```diff
@@ -0,0 +1,372 @@
...
+
+Aside from the above, we know that many people in the community are fans of YOASOBI's music, so we know those fans will be happy to see another YOASOBI chart in the Loved voting. If you want to see a good LN collaboration practice chart in Loved, or you just want to jam to YOASOBI, remember to give *Ano Yume o Nazotte* your support!
+
+---
+
+From [everyone working on Project Loved](https://loved.sh/contributors), we hope you enjoyed this month's selection, and are looking forward to the next! The favoured beatmaps will hit the [Loved beatmap listing](https://osu.ppy.sh/beatmapsets?s=loved) shortly after the polls end. So don't forget to stay hydrated and vote on your favourite beatmaps in order for that to happen!
+
+More info about Project Loved can be found on its [wiki page](/wiki/Community/Project_Loved) and [website](https://loved.sh/). Stop by the `#osu-loved` channel of the [osu!dev Discord server](https://discord.gg/ppy) if you have any questions, concerns, or are interested in helping out.
```
Comment (Blooshing) [inline news/2022-12-15-project-loved-december-2022.md:368]:
```Suggestion
More info about Project Loved can be found on its [wiki page](/wiki/Community/Project_Loved) and [website](https://loved.sh/). If there are any questions, comments, concerns, or you are interested in helping out, please stop by the `#osu-loved` channel in the [osu!dev Discord server](https://discord.gg/ppy).
```

**Review by Blooshing — CHANGES_REQUESTED** [review]

File: news/2022-12-15-project-loved-december-2022.md [hunk 7]
```diff
@@ -0,0 +1,372 @@
...
+
+[![](/wiki/shared/news/2022-12-15-project-loved-december-2022/1360992.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[wa. - Black Lotus](https://osu.ppy.sh/beatmapsets/1360992#osu) by [blixys](https://osu.ppy.sh/users/13015896)\
+200 BPM, 1:47 | 8.37★\
+*description written by [rHO](https://osu.ppy.sh/users/1629553)*
+
+If you don't know who [blixys](https://osu.ppy.sh/users/13015896) is, now is the time to meet them and their beautifully chaotic maps.
+
+Speaking in a language that mostly consists of heavily constructed rhythms, map structure and sliders, blixys shows us what they are capable of — capturing destruction and rebirth in their map *Black Lotus*. Using their mapping and object manipulation skills to the max, blixys successfully pulls off an jaw-dropping beatmap full of flowers and lotuses. I can hear the people saying *"Beatmaps are never that serious and/or poetic, rHO!"* comments already, but if you don't believe me, take a gander and you'll see what I'm talking about 😛
```
Comment (Blooshing) [inline news/2022-12-15-project-loved-december-2022.md:57]: Sorry, not sorry for the oxford comma
``` Suggestion
Speaking in a language that mostly consists of heavily constructed rhythms, map structure, and sliders, blixys shows us what they are capable of — capturing destruction and rebirth in their map *Black Lotus*. Using their mapping and object manipulation skills to the max, blixys successfully pulls off a jaw-dropping beatmap full of flowers and lotuses. I can hear the people saying *"Beatmaps are never that serious and/or poetic, rHO!"* comments already, but if you don't believe me, take a gander and you'll see what I'm talking about 😛
```

**Review by Blooshing — CHANGES_REQUESTED** [review]

File: news/2022-12-15-project-loved-december-2022.md (hunk 7 above)
Comment (Walavouchey) [reply news/2022-12-15-project-loved-december-2022.md:57]: news posts have more relaxed writing rules, including the oxford comma being optional

File: news/2022-12-15-project-loved-december-2022.md (hunk 4 above)
Comment (aceticke) [inline news/2022-12-15-project-loved-december-2022.md:120]:
```suggestion
\[Ekoro](https://osu.ppy.sh/users/284905)'s take on LIQU@.'s *Yotogibanashi no Kamikakushi* is very reminiscent of his own very popular map [UNDEAD CORPORATION - Everything will Freeze [\Time Freeze\]](https://osu.ppy.sh/beatmapsets/158023#osu/555797). Just like Time Freeze, this map features 240 BPM death/spaced streams, along with the iconic squares littered throughout the map. The main difference is that this song only sits at 1:48 in length. Plenty of room for some retrying, and world class score potential!
```

Comment (aceticke) [reply news/2022-12-15-project-loved-december-2022.md:120]: missing backslash on diff name which broke the embed

**Review by aceticke — CHANGES_REQUESTED** [review]
noticed a broken link fixed now

File: news/2022-12-15-project-loved-december-2022.md (hunk 7 above)
Comment (aceticke) [reply news/2022-12-15-project-loved-december-2022.md:57]: still would be good to fix an -> a but yeah

File: news/2022-12-15-project-loved-december-2022.md (hunk 6 above)
Comment (Hiviexd) [reply news/2022-12-15-project-loved-december-2022.md:368]: prefer current wording, we already used it many times too

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**